packages/cli/src/utils/build.js

```diff
@@ -159,7 +159,7 @@ const forceIncludeDumbPath = (appConfig, filePath) => {
   return (
     filePath.endsWith('package.json') ||
     filePath.endsWith('definition.json') ||
-    filePath.endsWith(path.join('bin', 'linux-x64-node-10', 'deasync.node')) ||
+    filePath.endsWith(path.join('bin', 'linux-x64-node-12', 'deasync.node')) ||
```

Comment:
Not sure if I understand [AWS Lambda runtime support policy](https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html) correctly. It seems Node.js 10 _could_ still be running after it is deprecated on May 28. Does that mean we should keep both Node 10 and 12 here?

Reply:
Good question! so my understanding of how this works is that functions will keep running, but at some point we won't be able to update them anymore. This has mostly been fine for us, since a lot of our functions are long-lived.

But, that does mean if we update boilerplates on godzilla apps anytime after may, we'll need to also bump their node versions (and by extension, their platform versions). though, I suppose we could be building a new boilerplate between merging this and upgrading node. It's probably safest to include as many node binaries as we can, I can't imagine they're that big. I can tweak this and the other spot.